source/infrastructure/lib/aws-instance-scheduler-stack.ts

```diff
@@ -15,22 +15,22 @@
  *****************************************************************************/
 
 import * as cdk from 'aws-cdk-lib';
+import {Aws, RemovalPolicy} from 'aws-cdk-lib';
```

Comment:
Can we be more consistent with the spacing of the destructuring syntax? Follow the pattern established in the file.